src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -77,16 +79,23 @@ public FindCommand parse(String args) throws ParseException {
                         PREFIX_LEAVE, PREFIX_DATE, PREFIX_HOURLYSALARY, PREFIX_HOURSWORKED, PREFIX_TAG,
                         PREFIX_OVERTIME);
 
-        String preamble = argMultimap.getPreamble();
+        String preamble = argMultimap.getPreamble().trim();
         if (trimmedArgs.isEmpty() && preamble.isEmpty()) {
+            // no arguments provided to the command
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
+        if (!preamble.isBlank()) { // a preamble exists
+            if (!preamble.equals(UNPAID_PREDICATE_KEYWORD)) {
+                throw new ParseException(
+                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+            }
+        }
```

Comment:
```suggestion
        if (!preamble.isBlank() && !preamble.equals(UNPAID_PREDICATE_KEYWORD) ) {  // preamble that is not 'unpaid'
                throw new ParseException(
                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
        }
```
Perhaps consider merging both tests together?

Reply:
Yup that sounds good!